### DISCUSSION TITLE
Cancellare e Anonimizzare le segnalazioni

### DISCUSSION THREAD

**Discussion by mfagiolihi-net** [globaleaks/globaleaks-whistleblowing-software#4143] [General]
Salve,
ho impostato una scadenza di 5 anni nei canali di segnalazione, mi potete confermare che dopo 5 anni le segnalazioni vengono cancellate?
Ho il dubbio che vengano cancellate anche segnalazioni che sono ancora aperte, magari perchè hanno generato una causa legale. Esiste un modo per cancellare solo le segnalazioni in stato "chiuso".

Esiste un registro delle segnalazioni che mantiene anche dopo i 5 anni un elenco delle segnalazioni anonimizzando alcuni campi?

Come si cancellano gli audit log? la versione Changes in version 4.15.0 dovrebbe avere implementato una funzione di cancellazione ma non trovo dove si configura

**Comment by evilaliv3** [comment]
Grazie per la domanda @mfagiolihi-net 

Al momento  le segnalazioni vengono cancellate indipendentemente dallo stato in cui si trovano perchè la piattaforma è stata pensata per progetti ad alto rischio di tipo giornalistico dove si vuole che anche se i riceventi non sono in grado di accedere (magari sono stati arrestati) le segnalazioni vengano cancellate.

Al momento non esiste un registro delle segnalazioni perchè si è pensato che qualunque informazione dovesse essere cancellata, ma certamente prevediamo di estendere questo aspetto preservando un sintetico registro.

In merito agli audit log non è possibile al moment ocancellarli e vengono cancellati automaticamente dopo 10 anni.  In future versioni prevediamo di rendere questo parametro configurabile.

Ci sono alcuni di questi aspetti su cui vorresti lavorare direttamente sviluppandoli o hai modo di supportare il progetto con qualche donazione così che si possa coinvolgere noi stessi nuovi sviluppatori?

**Reply by mfagiolihi-net** [reply]
Grazie @evilaliv3 per la risposta
Mi piacerebbe collaborare ma non ho molto tempo libero lavoro in una azienda
